streams/src/test/java/org/apache/kafka/streams/integration/ProcessingExceptionHandlerIntegrationTest.java

```diff
@@ -61,6 +61,8 @@
 public class ProcessingExceptionHandlerIntegrationTest {
     private final String threadId = Thread.currentThread().getName();
 
+    private final static Instant TIMESTAMP = Instant.now();
```

Comment:
```suggestion
    private static final Instant TIMESTAMP = Instant.now();
```

Reply:
@sebastienviale -- fixing checkstyle error.

Reply:
@mjsax thanks, I forgot to check before pushing